src/bootstrap/lib.rs

```diff
@@ -205,7 +205,8 @@ const EXTRA_CHECK_CFGS: &[(Option<Mode>, &'static str, Option<&[&'static str]>)]
     (
         Some(Mode::Std),
         "target_arch",
-        Some(&["asmjs", "spirv", "nvptx", "nvptx64", "le32", "xtensa"]),
+        // #[cfg(bootstrap)] loongarch64
```

Comment:
Do we need this comment?

Reply:
Yes. The release team wants it to do be able to remove code (here loongarch64) when they do the beta bump otherwise it will not be removed.

cf. https://rustc-dev-guide.rust-lang.org/building/new-target.html#adding-a-target-specification

Reply:
After I tried to delete " // #[cfg(bootstrap)] loongarch64", I found that the CI check also passed successfully. From the results of the tests run so far, it seems that it has no effect.

Reply:
I found in the link below it is mentioned that only libstd uses cfg (bootstrap).
https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#why-does-only-libstd-use-cfgbootstrap

Reply:
> After I tried to delete " // #[cfg(bootstrap)] loongarch64", I found that the CI check also passed successfully. From the results of the tests run so far, it seems that it has no effect.

There isn't any CI checks for this. I don't even know if it would be possible to have one because entries added here maybe be added for more than a cycle.



> I found in the link below it is mentioned that only libstd uses cfg (bootstrap). https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#why-does-only-libstd-use-cfgbootstrap

This is talking about the attribute `#[cfg(bootstrap)]` not the comment. T-release just does a grep on the entire repo and remove everything under `cfg(bootstrap)`.

----

TLDR please put it back so that T-release can remove it after the beta bump.

Reply:
@ Urgau  Thanks a lot. I've put it back.